R/RunDiagnostics.R

```diff
@@ -749,7 +762,9 @@ executeDiagnostics <- function(cohortDefinitionSet,
           useExternalConceptCountsTable = FALSE,
           incremental = incremental,
           conceptIdTable = "#concept_ids",
-          recordKeepingFile = recordKeepingFile
+          recordKeepingFile = recordKeepingFile,
+          useAchilles = useAchilles,
+          resultsDatabaseSchema = resultsDatabaseSchema
```

Comment:
I think this should be `achillesDatabaseSchema`

Reply:
changed.